src/ingest_llm_as/processors/conversation_ingestor.py

```
@@ -76,52 +80,73 @@ async def process_pending_conversations(self) -> int:
             )
             
-            # 2. Write to Obsidian
+            # 2. Write to Obsidian Vault (Durable local backup)
             file_path = await self._write_to_vault(source_id, platform, summary_content, captured_at)
             
             # 3. Generate Embedding (Vector)
-            # Combine content for embedding: platform + summary + first few messages
             conversation_text = f"Platform: {platform}\nSummary: {summary_content}\n"
-            for msg in messages[:5]: # Include context from first 5 msgs
-                conversation_text += f"{msg.get('role', '')}: {msg.get('content', '')}\n"
+            for msg in messages[:EMBEDDING_CONTEXT_MESSAGE_COUNT]: # Include context from first 5 msgs
```

Ruff (0.14.11)

Undefined name `EMBEDDING_CONTEXT_MESSAGE_COUNT`

(F821)